bindings/python/.evergreen/test.sh

```diff
@@ -34,6 +40,10 @@ elif [ "Darwin" = "$(uname -s)" ]; then
              "/Library/Frameworks/Python.framework/Versions/3.8/bin/python3"
              "/Library/Frameworks/Python.framework/Versions/3.9/bin/python3"
              "/Library/Frameworks/Python.framework/Versions/3.10/bin/python3")
+    export CSFLE_PATH="../csfle/lib/mongo_csfle_v1.dylib"
+    python3 drivers-evergreen-tools/.evergreen/mongodl.py --component csfle \
+    --version 6.0.0-rc4 --out ../csfle/
```

Comment:
It's confusing to do a multi-line shell command without indenting. Can you fix this here and throughout?:
```
    python3 drivers-evergreen-tools/.evergreen/mongodl.py --component csfle \
        --version 6.0.0-rc4 --out ../csfle/
```

Reply:
Done.